src/Monolog/Handler/SyslogUdpHandler.php

```diff
@@ -93,10 +94,13 @@ protected function makeCommonSyslogHeader(int $severity): string
             $hostname = '-';
         }
 
-        $date = $this->getDateTime();
```

Comment:
As you removed that function usage entirely I would say let's remove the function itself.

Reply:
Haha. Yes, of course... Done.